src/object/srv_obj_migrate.c

```diff
@@ -3434,6 +3435,13 @@ ds_obj_migrate_handler(crt_rpc_t *rpc)
 		D_GOTO(out, rc = -DER_INVAL);
 	}
 
+	ds_rebuild_running_query(migrate_in->om_pool_uuid, rebuild_ver);
```

Comment:
```suggestion
	ds_rebuild_running_query(migrate_in->om_pool_uuid, &rebuild_ver);
```

Reply:
Ah, yes. Sorry for stupid mistakes. Thanks